### DISCUSSION TITLE
How to output the debug log of go-fil-markets in Lotus

### DISCUSSION THREAD

**Discussion by s1mple1122** [filecoin-project/lotus#9137] [General Q&A] [answered]
log=DEBUG or DEBUG=true     cant output

**Comment by LexLuthr** [comment, marked as answer]
`$ lotus-miner log set-level --help
NAME:
   lotus-miner log set-level - Set log level

USAGE:
   lotus-miner log set-level [command options] [level]

DESCRIPTION:
   Set the log level for logging systems:
   
      The system flag can be specified multiple times.
   
      eg) log set-level --system chain --system chainxchg debug
   
      Available Levels:
      debug
      info
      warn
      error
   
      Environment Variables:
      GOLOG_LOG_LEVEL - Default log level for all log systems
      GOLOG_LOG_FMT   - Change output log format (json, nocolor)
      GOLOG_FILE      - Write logs to file
      GOLOG_OUTPUT    - Specify whether to output to file, stderr, stdout or a combination, i.e. file+stderr

OPTIONS:
   --system value  limit to log system  (accepts multiple inputs)`
   
   $ lotus-miner log list | grep mark
market_adapter
markets
retrievalmarket_impl
storagemarket_network
storagemarket_impl
markets-rtvl
markets-rtvl-reval
